interview-questions.md

```diff
@@ -9,59 +9,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The term "this" is used when youre calling on a variable that is nesting within a component. Kinda like calling within it self. 
 
-  Researched answer:
+  Researched answer: after some research, im even more confused 
```

Comment:
Yes, `this` is special. But in general, it is a keyword. You cannot change the value of `this`. In JavaScript, the `this` keyword refers to an object. Checkout w3schools for an explanation. https://www.w3schools.com/js/js_this.asp